platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java

```diff
@@ -21,7 +21,7 @@
 import com.swirlds.config.api.ConfigProperty;
 
 /**
- * Settimgs for the benchmarks
+ * Settings for the benchmarks
```

Comment:
This is like the 2nd time in this PR I've had to really look at it to see what's changed.  Is it the font, or just my eyesight?